cumulusci/cli/tests/test_org.py

```diff
@@ -403,16 +410,98 @@ def test_org_import__persistent_org(self, cmd):
             stderr=io.BytesIO(b""), stdout=io.BytesIO(result), returncode=0
         )
 
+        # Mock the call to get the Organization sObject
+        responses.add(
+            method="GET",
+            url="https://instance/services/data",
+            status=200,
+            json=[{"version": "54.0"}],
+        )
+        responses.add(
+            method="GET",
+            url="https://instance/services/data/v54.0/sobjects/Organization/OODxxxxxxxxxxxx",
+            json={
+                "TrialExpirationDate": None,
+                "OrganizationType": "Developer Edition",
+                "IsSandbox": True,
+                "InstanceName": "CS420",
+                "NamespacePrefix": None,
+            },
+            status=200,
+        )
+
         out = []
-        with mock.patch("click.echo", out.append), pytest.raises(
-            click.UsageError, match="cci org connect"
-        ):
+        with mock.patch("click.echo", out.append):
             run_click_command(
                 org.org_import,
                 username_or_alias="test@test.org",
                 org_name="test",
                 runtime=runtime,
             )
+            runtime.keychain.set_org.assert_called_once()
+            created_org = runtime.keychain.set_org.call_args[0][0]
```

Comment:
I think something like this allows us to get rid of line 442:
```suggestion
            runtime.keychain.set_org.assert_called_once_with(created_org, global_org=mock.ANY, save=mock.ANY)
```